### PR TITLE
Docs - Update docs for monitor.

### PR DESCRIPTION
**Pull request by guoshzhao**
**Description**
Update docs for monitor.

### CODE REVIEW

File: docs/user-tutorial/monitor.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+id: monitor
+---
+
+# Monitor
+
+SuperBench has realized a simple `Monitor` module to collect the system metrics during the benchmarking. Currently only support CUDA platform. User can enable it in the config file.
```
Comment (cp5555) [inline docs/user-tutorial/monitor.md:7]: SuperBench provides a `Monitor` module to collect the system metrics and detect the failure during the benchmarking. Currently this monitor supports CUDA platform only. Users can enable it in the config file.

**Review by cp5555 — APPROVED** [review]

File: docs/user-tutorial/monitor.md (hunk 1 above)
Comment (guoshzhao) [reply docs/user-tutorial/monitor.md:7]: Thanks

File: docs/user-tutorial/monitor.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+id: monitor
+---
+
+# Monitor
+
+SuperBench provides a `Monitor` module to collect the system metrics and detect the failure during the benchmarking. Currently this monitor supports CUDA platform only. Users can enable it in the config file.
+
+## Configuration
```
Comment (abuccts) [inline docs/user-tutorial/monitor.md:9]: move it to superbench-config.mdx?

Comment (guoshzhao) [reply docs/user-tutorial/monitor.md:9]: superbench-config.mdx has already contained the Configuration part.

**Review by abuccts — APPROVED** [review]

**Pull request merged**